# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../smpeg/gcc41.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur smpeg-2003-05-09-orig/MPEGaudio.h smpeg-2003-05-09/MPEGaudio.h
--- smpeg-2003-05-09-orig/MPEGaudio.h	2001-07-17 19:52:24.000000000 +0000
+++ smpeg-2003-05-09/MPEGaudio.h	2006-04-02 17:37:38.997298250 +0000
@@ -369,4 +369,10 @@
   double timestamp[N_TIMESTAMPS];
 };
 
+    void Play_MPEGaudioSDL(void *udata, Uint8 *stream, int len);
+    int Play_MPEGaudio(MPEGaudio *audio, Uint8 *stream, int len);
+#ifdef THREADED_AUDIO
+    int Decode_MPEGaudio(void *udata);
+#endif
+
 #endif /* _MPEGAUDIO_H_ */
